app/src/main/java/org/xbmc/kore/ui/AddonDetailsFragment.java

```diff
@@ -225,70 +235,39 @@ public void onSuccess(String result) {
             }
         }, callbackHandler);
     }
 
-    /**
-     * Display the addon details
-     *
-     * @param addonDetails Addon details
-     */
-    private void displayAddonDetails(AddonType.Details addonDetails) {
-        mediaTitle.setText(addonDetails.name);
-        mediaUndertitle.setText(addonDetails.summary);
-
-        mediaAuthor.setText(addonDetails.author);
-        mediaVersion.setText(addonDetails.version);
-
-        mediaDescription.setText(addonDetails.description);
-
-        // Images
+    private void setImages(String poster, String fanart) {
         Resources resources = getActivity().getResources();
         DisplayMetrics displayMetrics = new DisplayMetrics();
         getActivity().getWindowManager().getDefaultDisplay().getMetrics(displayMetrics);
-
         int artHeight = resources.getDimensionPixelOffset(R.dimen.now_playing_art_height),
                 artWidth = displayMetrics.widthPixels;
-        if (!TextUtils.isEmpty(addonDetails.fanart)) {
-            int posterWidth = resources.getDimensionPixelOffset(R.dimen.addondetail_poster_width);
-            int posterHeight = resources.getDimensionPixelOffset(R.dimen.addondetail_poster_heigth);
-            mediaPoster.setVisibility(View.VISIBLE);
-            UIUtils.loadImageIntoImageview(hostManager,
-                    addonDetails.thumbnail,
-                    mediaPoster, posterWidth, posterHeight);
-            UIUtils.loadImageIntoImageview(hostManager,
-                    addonDetails.fanart,
-                    mediaArt, artWidth, artHeight);
-        } else {
-            // No fanart, just present the poster
-            mediaPoster.setVisibility(View.GONE);
-            UIUtils.loadImageIntoImageview(hostManager,
-                    addonDetails.thumbnail,
-                    mediaArt, artWidth, artHeight);
-            // Reset padding
-            int paddingLeft = mediaTitle.getPaddingRight(),
-                    paddingRight = mediaTitle.getPaddingRight(),
-                    paddingTop = mediaTitle.getPaddingTop(),
-                    paddingBottom = mediaTitle.getPaddingBottom();
-            mediaTitle.setPadding(paddingLeft, paddingTop, paddingRight, paddingBottom);
-            mediaUndertitle.setPadding(paddingLeft, paddingTop, paddingRight, paddingBottom);
-        }
+        int posterWidth = resources.getDimensionPixelOffset(R.dimen.addondetail_poster_width);
+        int posterHeight = resources.getDimensionPixelOffset(R.dimen.addondetail_poster_height);
+
+        UIUtils.loadImageIntoImageview(hostManager,
+                                       TextUtils.isEmpty(fanart)? poster : fanart,
+                                       mediaArt, artWidth, artHeight);
+        UIUtils.loadImageIntoImageview(hostManager,
+                                       poster,
+                                       mediaPoster, posterWidth, posterHeight);
 
-        setupEnableButton(addonDetails.enabled);
```

Comment:
You can't remove this call. It is used to show if the addon is enabled or not when the user enters the screen, so it is needed.